pkg/metadata/host/host_test.go

```diff
@@ -113,3 +117,51 @@ func TestGetLogsMeta(t *testing.T) {
 	meta = getLogsMeta()
 	assert.Equal(t, &LogsMeta{Transport: "HTTP"}, meta)
 }
+
+func TestGetInstallMethod(t *testing.T) {
+	dir, err := ioutil.TempDir("", "test_install_method")
+	assert.Nil(t, err)
+	defer os.RemoveAll(dir)
+
+	installInfoPath := path.Join(dir, "install_info")
+
+	// ------------- Without file, the install is considered private
+	installMethod := getInstallMethod(installInfoPath)
+	require.Equal(t, "private", installMethod.Name)
+	require.Equal(t, "n/a", installMethod.Tool)
+	require.Equal(t, "n/a", installMethod.Version)
+
+	// ------------- with a correct file
+	var installInfoContent = `
+---
+install_method:
+  name: chef-15
+  tool: chef
+  version: chef-4.2.1
```

Comment:
Why do we need 3 different things here?

Also, the third version should maybe be `chef-cookbook-4.2.1` to not be confused with the first one.

Reply:
I thought about filtering by tool in dashboards but it's true we could parse it agent-side when reading the name.

And I'll update the chef-cookbook accordingly.